ckanext/datapusher/helpers.py

```diff
@@ -29,3 +29,9 @@ def datapusher_status_description(status: dict[str, Any]):
         return captions.get(status['status'], status['status'].capitalize())
     else:
         return _('Not Uploaded Yet')
+
+
+def is_datapusher_format(resource_format: str):
```

Comment:
This helper could be expanded to cover the other two checks, `res.datastore_active` and `res.url_type not in h.datastore_rw_resource_url_types()`, since they're always used together.

Reply:
Helpers and conditions combined into one helper.